js/checkboxgroup/index.svelte

```diff
@@ -9,7 +9,7 @@
 	export let visible = true;
 	export let value: string[] = [];
 	export let value_is_output = false;
-	export let choices: string[];
+	export let choices: string[][];
```

Comment:
I think this would be a better type to indicate it can only be tuples:  `[string, string | number][]`

Reply:
Nice thanks